docs/ko/docs/Benchmarks.md

```diff
@@ -0,0 +1,34 @@
+# 벤치마크
+
+독립적인 TechEmpower 벤치마크에 따르면 Uvicorn에서 실행되는 **FastAPI** 애플리케이션을 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=로 표시합니다. zijzen-7" class="external-link" target="_blank">사용 가능한 가장 빠른 Python 프레임워크 중 하나</a>로  Starlette 및 Uvicorn보다 낮습니다. (FastAPI에서 내부적으로 사용). (*)
+
+그러나 벤치마크를 확인하고 비교할 때 다음 사항을 염두에 두어야 합니다.
+
+## 벤치마크 및 속도
+
+벤치마크를 확인할 때 서로 다른 유형의 여러 도구를 동등하게 비교하는 것이 일반적입니다.
+
+특히 Uvicorn, Starlette 및 FastAPI를 함께 비교하는 것을 확인하기 위해서(다른 많은 도구 중에서).

```

Comment:
```suggestion
특히 (다른 많은 도구 중에서) Uvicorn, Starlette 및 FastAPI를 함께 비교하는 것을 확인하기 위한 방법이 필요합니다.
```

I think it would be better to complete the sentence by the context. The next sentence refers to solution of the problem of comparing several tools. So it the previous sentence-this sentence-might be completed like `방법이 필요합니다.`